src/code42cli/cmds/securitydata.py

```diff
@@ -97,7 +98,25 @@
 file_category_option = click.option(
     "--file-category",
     multiple=True,
-    type=click.Choice(list(FileCategory.choices())),
+    type=MapChoice(
+        choices=list(FileCategory.choices()),
+        extras_map={
+            "AUDIO": "Audio",
```

Comment:
If these mapping are already in the latest py42, you could use the actual constants
`FileCategory.DOCUMENT` etc

Reply:
We actually may need a release in order for that to happen

Reply:
Ah, true. Then if (god forbid) they ever change again, it'll just work (although we'd need a new set of mappings for the now changed-again values).

Reply:
we do need a version bump / new py42 release, yeah -- that's why the tests are failing on this PR.